### PR TITLE
FIR-33074 implementation of Connection.getClientInfo()

### PR DESCRIPTION
**Pull request by alexradzin**

### CODE REVIEW

File: src/test/java/com/firebolt/jdbc/connection/FireboltConnectionTest.java [hunk 1]
```diff
@@ -651,5 +649,30 @@ void unsupportedNativeSql() throws SQLException {
 		}
 	}
 
+	@Test
+	void getClientInfo() throws SQLException {
+		try (FireboltConnection connection = createConnection(URL, connectionProperties)) {
+			Properties info = connection.getClientInfo();
+			System.out.println(info);
```
Comment (ptiurin) [inline src/test/java/com/firebolt/jdbc/connection/FireboltConnectionTest.java:656]: Let's remove this print before merging

**Review by ptiurin — COMMENTED** [review]
Otherwise looks good

File: src/main/java/com/firebolt/jdbc/connection/settings/FireboltSessionProperty.java [hunk 2]
```diff
@@ -1,106 +1,118 @@
...
 
 	private final String key;
 	private final Object defaultValue;
 	private final Class<?> clazz;
 	private final String description;
+	private final Function<FireboltProperties, Object> valueGetter;
 	private final String[] aliases;
+	private static final Map<String, FireboltSessionProperty> aliasToProperty = Arrays.stream(values())
+			.flatMap(p -> Stream.concat(Stream.of(p.key), Arrays.stream(p.aliases)).map(a -> Map.entry(a, p)))
+			.collect(toMap(Entry::getKey, Entry::getValue, (o, t) -> t, () -> new TreeMap<>(CASE_INSENSITIVE_ORDER)));
```
[the remark this answers is not in the document]
Comment (alexradzin) [reply src/main/java/com/firebolt/jdbc/connection/settings/FireboltSessionProperty.java]: Extracted logic to static functions, so now this line is shorter and hopefully clearer.

**Review by ptiurin — APPROVED** [review]
lgtm

**Pull request merged**